go/client/cmd_chat_send.go

```diff
@@ -74,8 +81,20 @@ func (c *cmdChatSend) Run() (err error) {
 		conversationInfo = *resolved
 	}
 
-	if err = c.G().UI.GetTerminalUI().PromptForConfirmation(fmt.Sprintf("Send to %s?", conversationInfo.TlfName)); err != nil {
-		return err
+	switch {
+	case userChosen && len(c.message) == 0:
+		c.message, err = c.G().UI.GetTerminalUI().Prompt(PromptDescriptorEnterChatMessage, fmt.Sprintf("Send to [%s]? Hit Ctrl-C to cancel, or enter message content to send: ", conversationInfo.TlfName))
+		if err != nil {
+			return err
+		}
+	case userChosen:
+		return errors.New("\"this should never happen\"")
```

Comment:
That was supposed to be a joke; but I'll switch it out to be more descriptive :D